hera/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:14.04
+MAINTAINER John Vivian, jtvivian@gmail.com
+
+# install dependencies first
+RUN apt-get update  && apt-get install -y \
+		build-essential \
+		liblzma-dev \
+		libbz2-dev \
+		libz-dev \
+		git \
+		wget
+
+
+# Install cmake 3.9.1
+WORKDIR /
+# git clone hera since we're here
+RUN git clone https://github.com/bioturing/hera.git
+# On with cmake
+RUN wget https://cmake.org/files/v3.9/cmake-3.9.1.tar.gz
+RUN tar -xzvf cmake-3.9.1.tar.gz
+WORKDIR /cmake-3.9.1
```

Comment:
We've got multiple images that install cmake, can we factor that out into a separate image we use? I'm fine if you open an issue for later and don't make that change in this PR.

Reply:
Great idea, I'll make that now and rebase